judgels-frontends/raphael/src/modules/api/sandalphon/problemBundle.ts

```diff
@@ -35,6 +35,7 @@ export interface Item {
   number?: number;
   meta: string;
   config: ItemConfig;
+  reasonNotAllowedToSubmit?: string;
```

Comment:
We never use the actual string for BUNDLE contest, also it will be the same for all items. For memory efficiency let's just pass down `disabled` instead of the actual reason string